aws_lambda_builders/workflows/nodejs_npm_esbuild/esbuild.py

```diff
@@ -154,15 +133,86 @@ def build_esbuild_args_from_config(self) -> "EsbuildCommandBuilder":
         """
         args = []
 
-        args.extend(self._get_boolean_args())
-        args.extend(self._get_single_value_args())
-        args.extend(self._get_multi_value_args())
+        for config_key, config_value in self._bundler_config.items():
+            if config_key in NON_CONFIGURABLE_VALUES:
+                continue
```

Comment:
Worth logging a warning message so that user will be aware that their setting is not applied as they wanted to. We might want to provide detailed steps for them to fix the issue on their side.